resources/examples/Showtimes/Controllers/Movie.php

```diff
@@ -118,6 +118,7 @@ public function PATCH()
      * @api-uriSegment {/movies/+id} id (integer) - Movie ID
      *
      * @api-contentType application/json
+     * @api-capability DELETE_CONTENT
```

Comment:
On second thought, other capabilities show this off just fine, so :man_shrugging: